Lesson39/savas/src/components/__test__/DialogTest.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import DialogTest from "../DialogTest";
+
+test("When a button is clicked, a dialog should show up on a screen", () => {
```

Comment:
Add another test to test that the dialog can be closed correctly

Reply:
I couldn't do it similar method, but I searched it and found 'waitFor' event. And it works :)